Resources/views/Collector/execute.html.twig

```diff
@@ -0,0 +1,43 @@
+{% set nbVisibleCols = 6 %}
+<p>
+{% for label in data[0]|keys %}
+    <a class="btn btn-sm" style="padding: 3px; margin-bottom: 3px;" onclick="toggleColumn(this, 'query-results-{{ id }}', '{{ loop.index }}')">
+        <span class="{{ loop.index <= nbVisibleCols ? "" : "text-muted" }}">
+            {{ label }}
+        </span>
+    </a>
+{% endfor %}
+</p>
+
+{% if data[0]|length > 1 %}
+    <table style="margin: 5px 0;" id="query-results-{{ id }}">
+        <thead>
+        <tr>
+            {% for label in data[0]|keys %}
+                <th class="col col{{ loop.index }} {{ loop.index <= nbVisibleCols ? "" : "hidden" }}">{{ label }}</th>
+            {% endfor %}
+        </tr>
+        </thead>
+        <tbody>
+        {% for row in data %}
+            <tr>
+                {% for key, item in row %}
+                    <td class="col col{{ loop.index }} {{ loop.index <= nbVisibleCols ? "" : "hidden" }}">{{ item }}</td>
+                {% endfor %}
+            </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+{% else %}
+    <pre style="margin: 5px 0;">
+        {%- for row in data -%}
+            {{ row|first }}{{ "\n" }}
+        {%- endfor -%}
+    </pre>
```

Comment:
I don't think this makes sense here. Displaying results as a text is not something which makes sense in the general case (it does only for queries optimized for that, like the postgresql explain command) and it would break things for queries selecting a single column (for instance ``SELECT email FROM users`` to get the list of all used emails)